qiskit/extensions/standard/ch.py

```diff
@@ -73,6 +68,13 @@ def inverse(self):
         """Invert this gate."""
         return CHGate()  # self-inverse
 
+    def to_matrix(self):
+        """Return a Numpy.array for the Ch gate."""
+        return numpy.array([[1, 0, 0, 0],
+                            [0, 1, 0, 0],
+                            [0, 0, 0.707, 0.707],
+                            [0, 0, 0.707, -0.707]], dtype=complex)
+
```

Comment:
We should verify the indexing here, and make sure to get enough precision to pass e.g. `np.isclose`.

Reply:
@kdk I fixed this and added a test. Thanks for the good catch.